src/nl/rubensten/texifyidea/window/EquationPreviewToolWindow.kt

```diff
@@ -0,0 +1,14 @@
+package nl.rubensten.texifyidea.window
+
+import javax.swing.*
+
+
+class EquationPreviewToolWindow(val equation_text:String) {
```

Comment:
Add spacing after `:`, in this case `equationText: String`.

Also, rename to `equationText`.